pace.robot

```diff
@@ -1,30 +1,42 @@
+# Tip: Everything that starts with # is a guidance for you and will not get executed.
+
 *** Settings ***
-# Don't change these settings when trying out Qentinel Pace for the first time.
-Documentation           Test suites for Pace starter.
+
+Documentation           Test suite for Pace starter.
 Library                 QWeb
 Suite Setup             Open Browser    about:blank    chrome
 Suite Teardown          Close All Browsers
 
+# Tip: The Settings section is where you define test suite level configuration.
+#      Suite Setup and Teardown are actions that happen before and after a test suite run.
+#      For first time users, we don't recommend to change them.
```

Comment:
Apologies for going grammar nazi, but this would probably read better as "we don't recommend **changing these**".